### ISSUE TITLE
Implement new Triggers - "dsl_ode_trigger_latest_new" and "dsl_ode_trigger_earliest_new"

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#576]
*updated 8/9/21 - with name change from "Newest" and "Oldest" to "Latest" and "Earliest"* 

Implement two new Triggers to trigger on the "Newest" and "Oldest" objects within a frame.
```C
/**
 * @brief Latest Trigger that checks for the persistence of Objects tracked 
 * and will trigger on the Object with the least time of persistence (latest)
 * if at least one is found.
 * @param[in] name unique name for the ODE Trigger
 * @param[in] source unique source name filter for the ODE Trigger, NULL = ANY_SOURCE.
 * @param[in] class_id class id filter for this ODE Trigger.
 * @param[in] limit limits the number of ODE occurrences, a value of 0 = NO limit.
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_ODE_TRIGGER_RESULT otherwise.
 */
DslReturnType dsl_ode_trigger_latest_new(const wchar_t* name, 
    const wchar_t* source, uint class_id, uint limit);

/**
 * @brief Earliest Trigger that checks for the persistence of Objects tracked 
 * and will trigger on the Object with the greatest time of persistence (earliest) 
 * if at least one is found.
 * @param[in] name unique name for the ODE Trigger.
 * @param[in] source unique source name filter for the ODE Trigger, NULL = ANY_SOURCE.
 * @param[in] class_id class id filter for this ODE Trigger.
 * @param[in] limit limits the number of ODE occurrences, a value of 0 = NO limit
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_ODE_TRIGGER_RESULT otherwise.
 */
DslReturnType dsl_ode_trigger_earliest_new(const wchar_t* name, 
    const wchar_t* source, uint class_id, uint limit);

```

**Comment by shuli163love** [comment]
Hi. The demo videos in youtube show the usage of DSL, and the IVA Studio is similar to a engine editor of the game engine, which is a nice tool. So, do you have a plan to open the source of the IVA Studio? Thanks.

**Comment by rjhowell44** [comment]
Hi @shuli163love, perhaps in the future once DSL has been officially released, but not at this time.

**Comment by rjhowell44** [comment]
Merged into v0.19.alpha
